bindata/network/ovn-kubernetes/001-crd.yaml

```diff
@@ -262,3 +262,195 @@ status:
+                          master:
+                            description: Name of the master inteface. Usually used for macvlan and ipvlan.
+                              Need not be specified if it can be inferred from the IP address
+                            type: string
+                        type: object
+                      ip:
+                        description: IP configuration arguments.
+                        properties:
+                          addresses:
+                            description: IP addresses to configure on the interface.
+                            items:
+                              type: string
+                            type: array
+                          gateway:
+                            description: IP address of the next-hop gateway, if it cannot be automatically determined.
+                            type: string
+                          destinations:
+                            description: List of CIDR blocks that the pod is allowed to connect to via this interface.
+                              If not provided, the pod can connect to any destination.
+                            items:
+                              type: string
+                            type: array
+                        required:
+                        - addresses
+                        type: object
+                      podIP:
+                        description: List of Per-pod IP configuration.
+                        items:
+                          description: Each item has the name of a pod (possibly ending with "*"), list of ip addresses
+                            to configure the interface, a gateway ip and a list of destinations.
+                          properties:
+                            name:
+                              description: Name of a pod (possibly ending with "*").
+                              type: string
+                            addresses:
+                              description: IP addresses to configure on the interface.
+                              items:
+                                type: string
+                              type: array
+                            gateway:
+                              description: IP address of the next-hop gateway, if it cannot be automatically determined.
+                              type: string
+                            destinations:
+                              description: List of CIDR blocks that the pod is allowed to connect to via this interface.
+                                If not provided, the pod can connect to any destination.
+                              items:
+                                type: string
+                              type: array
+                          required:
+                          - name
+                          - addresses
+                          type: object
+                        type: array
+                      ipConfig:
```

Comment:
remove `ipConfig` and `podIP` for now. Daniel is focussing on `ip` for the beginning. We can revisit this.